app/controllers/line_items_controller.rb

```diff
@@ -0,0 +1,60 @@
+class LineItemsController < ApplicationController
```

Comment:
I don't think this controller should be needed. Please list down what each action of this controller does.

Reply:
fixed